CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -483,9 +483,15 @@ end
 
 --! Called at the end of each day.
 function PlayerHospital:onEndDay()
-  -- Advise the player.
   if self:hasStaffedDesk() then
+    -- Are we providing advice today?
     self:dailyAdviceChecks()
+
+    -- Can we take our first patient?
+    if self.opened and self.num_visitors == 0 then
+      -- Give player a patient so they don't wait too long
+      self.world:spawnPatient(self)
+    end
   end
```

Comment:
@tobylane if the daily advisor checks could instead be tested against the below would that work?
```lua
function Hospital:canAcceptPatients()
  return self:hasStaffedDesk() and self.opened
end
```
(currently we just statically wait 5 months, regardless if the player is in motion or not)